po/de.po

```diff
@@ -22753,24 +22727,20 @@ msgid "No branch name specified"
 msgstr "Kein Branchname spezifiziert"
 
 #: builtin/stash.c:808
-#, fuzzy
 msgid "failed to parse tree"
-msgstr "Fehler beim Parsen von %s."
+msgstr "Parsen der Tree-Objekte fehlgeschlagen"
 
 #: builtin/stash.c:819
-#, fuzzy
 msgid "failed to unpack trees"
-msgstr "Fehler beim Entpacken des Tree-Objektes von HEAD."
+msgstr "Entpacken der Tree-Objekte fehlgeschlagen"
```

Comment:
des Tree-Objektes

Reply:
https://github.com/ruester/git-po-de/wiki/Translation-Guidelines
da es ja "trees" sind, würde ich da bei der Mehrzahl bleiben wollen... oder vielleicht eine andere Übersetzung für "trees" nehmen?

Reply:
Ja, sind mehrere gemeint, mein Fehler. Is OK so, Tree als Typ eines Objektes.